ui-new/html/waiting.html

```diff
@@ -0,0 +1,12 @@
+<section class="viewport waiting_state cash-out-color fiat-side" data-tr-section="waiting">
```

Comment:
On this case there are two different screens for cash_in and cash_out, the other is called cash_in_waiting_state. They should get merged into one, but at this point it would be faster to just replicate this over there.